### PR TITLE
Create bucket for saving terraform state of project resources

### PR DESCRIPTION
**Pull request by askoriy**
It is first part of change.
Next will be tag merged commit as new release, than change project factory to call new version of module and pass additional variables to it

### CODE REVIEW

**Comment by askoriy** [comment]
Don't merge now, seem there is a bug in google's project factory that brings bucket can't be created in newly created project.

File: modules/project/outputs.tf [hunk 1]
```diff
@@ -34,3 +34,8 @@ output secret_manager_service_account_private_key_encoded {
   value       = module.secret_manager_sa.private_key_encoded
   sensitive   = true
 }
+
+output terraform_state_bucket {
+  description = "Bucket for saving terraform state of project resources"
+  value       = module.project_factory.project_bucket_self_link
+}
```
Comment (sasjo) [inline modules/project/outputs.tf]: Add missing LF on last line.

File: modules/project/vars.tf [hunk 2]
```diff
@@ -35,6 +35,19 @@ variable default_service_account {
   default     = "deprivilege"
 }
 
+## Terraform state bucket
+variable "tribe" {
```
Comment (sasjo) [inline modules/project/vars.tf]: Follow the same pattern as for existing vars and remove the quotes around the name.

**Review by sasjo — CHANGES_REQUESTED** [review]
See comments

**Review by anakovt — APPROVED** [review]

**Pull request merged**